tests/default_missing_vals.rs

```diff
@@ -68,6 +68,7 @@ fn opt_present_with_value() {
 //             .default_missing_value("always")
 //         )
 //         .try_get_matches_from(vec!["", "--color="]);
+//     r.unwrap();
```

Comment:
?

Reply:
Yuck sorry, I added this line while I was briefly playing with this test. It has a fixme above it and I was wondering if this patch would fix it. Turns out no 😢 . L

I'll remove this line again.